src/platform/tests/TestPlatformMgr.cpp

```diff
@@ -59,12 +59,8 @@ static void TestPlatformMgr_StartEventLoopTask(nlTestSuite * inSuite, void * inC
 static void TestPlatformMgr_TryLockChipStack(nlTestSuite * inSuite, void * inContext)
 {
     bool locked = PlatformMgr().TryLockChipStack();
-    NL_TEST_ASSERT(inSuite, locked);
-
-    PlatformMgr().LockChipStack();
-    locked = PlatformMgr().TryLockChipStack();
-    PlatformMgr().UnlockChipStack();
-    NL_TEST_ASSERT(inSuite, locked);
+    if (locked)
```

Comment:
should we have a NL_TEST_ASSERT on locked being true ? Previous code was testing things (but double for recursive support). Should we keep at least one?

Reply:
This isn't possible because the lock is contended during this tests (by the events thread). By definition trylock isn't guaranteed to take the lock.

That's the cause of the flake in the bug this is closing.